recipes/allennlp/meta.yaml

```diff
@@ -0,0 +1,122 @@
+{% set name = "allennlp" %}
+{% set version = "0.4.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "dbc0b7de268a14ad978ad29ceb785c0a22cab13e2786f04fe933b4c71e9ad7aa" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  # Fix this when the new release is officially out.
+  url: https://github.com/allenai/allennlp/archive/v0.4.3.tar.gz
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  skip: True  # [win or py>=36]
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
```

Comment:
Would remove the explicit versions from the `python`s here. `skip` should be sufficient to get the intended behavior. These will likely complicate matters.